src/data/faq_de.json

```diff
@@ -532,14 +532,14 @@
                         ]
                     },
                     {
-                        "title": "Muss ich die App immer geöffnet lassen, damit Begegnungen aufgezeichnet werden?",
+                        "title": "Funktioniert die Aufzeichnung der Begegnungen auch, wenn ich die App schließe?",
                         "anchor": "close_app",
                         "active": true,
                         "textblock": [
-                            "Nein. Solange Sie in der App die Risiko-Ermittlung aktiviert und Bluetooth (und bei Android den Standort) eingeschalten haben, werden Begegnungen aufgezeichnet. Sie können die App schließen, sowohl durch Minimieren als auch durch 'Swipe-to-quit'.",
-                            "Sicherheitshalber öffnen Sie die App täglich nach Ablauf von 24 Stunden einmal. Weitere Hinweise finden Sie unter <a href='https://www.coronawarn.app/de/#howto' target='_blank' rel='noopener noreferrer'>So funktioniert die App am besten</a>.",
+                            "Ja. Solange Sie in der App die Risiko-Ermittlung aktiviert und Bluetooth (und bei Android die Standort-Dienste) eingeschalten haben, werden Begegnungen vom Exposure Notfication Framework von iOS/Android aufgezeichnet. Die App selbst muss dafür nicht geöffnet bleiben. Sie können die App schließen, sowohl durch Minimieren als auch durch 'Swipe-to-quit'.",
+                            "Sicherheitshalber können Sie die App einmal täglich öffnen. Weitere Hinweise finden Sie unter <a href='https://www.coronawarn.app/de/#howto' target='_blank' rel='noopener noreferrer'>So funktioniert die App am besten</a>.",

```

Comment:
Suggest changing to relative reference
href='https://www.coronawarn.app/de/#howto' =>
href='../#howto'

Reply:
fixed